### ISSUE TITLE
Itatic not working on MacOS

### ISSUE THREAD

**Issue by ray-x** [neovide/neovide#955]
**Describe the bug**
I tried different font set with Itatic, none of them work.

**To Reproduce**
1. set guifont=VictorMono\ Nerd\ Font:h18
2. use a colorscheme has itatic 
3. add comment
![image](https://user-images.githubusercontent.com/1681295/133113130-9cd07005-d104-4de2-b982-4c2784d5f1e1.png)

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS:   macOS 11.4
 - Neovide Version  HEAD
 - Neovim Version HEAD

**Please run `neovide --log` and paste the contents of the `.log` file here:**
[neovide_rCURRENT.log](https://github.com/neovide/neovide/files/7155494/neovide_rCURRENT.log)

**Additional context**

**Comment by PyGamer0** [comment]
Do you jave the italic version of the Victor Mono? Does it work in a terminal?

**Comment by ray-x** [comment]
Yes. Italic loads correctly in `kitty`, also autoloaded in `goneovim`

**Comment by huuson235** [comment]
I have the same problem on windows. I use Fira code NF.

**Comment by PyGamer0** [comment]
> I have the same problem on windows. I use Fira code NF.

Fira Code doesn't have italics right?

**Comment by LollipopFt** [comment]
> Fira Code doesn't have italics right?

it does.

**Comment by LollipopFt** [comment]
windows: using fira code, terminal and goneovim has italics, while neovide does not.

**Comment by dineshKumar777** [comment]
@ray-x I downloaded victormono NF from https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/VictorMono/Regular/complete
But italics not working in nvim-qt itself.. But at the same time normal VictorMono font displayed with Italics.


Can you build latest and give it a try. !!

**Comment by ray-x** [comment]
VictorMono still not working.
Also tried GoMono NF and I saw this after I set guifont = GoMono\ Nerd\ Font:h18

![image](https://user-images.githubusercontent.com/1681295/145562721-4691313c-320e-47cb-9993-379b76223811.png)

Seems it reset font to itatic only.

**Comment by dineshKumar777** [comment]
@ray-x 
baseline is nvim-qt.. pls verify whether italics are working in nvim-qt !
- go to nvim-qt
- command set guifont=*
- in popup select the font and apply
- in commandline, type :echo guifont
- this will give exact command which you can use here in neovide.

**Comment by ray-x** [comment]
nvim-qt
Itatic supported
If I run `Guifont` The result is `VictorMono Nerd Font:h18`

**Comment by fredizzimo** [comment]
I'm pretty sure this is fixed, but please re-open if it isn't
